update_ip/ip_getters/getters.py

```diff
@@ -1,24 +1,45 @@
-from update_ip.ip_getters.base import BaseIpGetter,  get_ip_from_http
+from update_ip.ip_getters.base import BaseIpGetter
 
 
 class DynDns(BaseIpGetter):
-    NAME= "dyndns.org"
-    URL= "http://checkip.dyndns.org"
+    NAME = "dyndns.org"
+    URL = "http://checkip.dyndns.org"
 
 
-'''
-class IpChicken(BaseIpGetter):
-    NAME= "ipchicken.com"
-    URL= "http://ipchicken.com/"
-'''
-
 class IfconfigMe(BaseIpGetter):
-    NAME= "ifconfig.me"
-    URL= "http://ifconfig.me/ip"
+    NAME = "ifconfig.me"
+    URL = "http://ifconfig.me/ip"
+
+
+class IPEchoNet(BaseIpGetter):
+    NAME = "ipecho.net"
+    URL = "http://ipecho.net/plain"
+
+
+class IP4Me(BaseIpGetter):
+    NAME = "ip4.me"
+    URL = "http://ip4.me/"
+
+
+class WhatIsMyPublicIP(BaseIpGetter):
+    NAME = "whatismypublicip.com"
+    URL = "http://www.whatismypublicip.com/"
+
+
+class WhatIsMyIP(BaseIpGetter):
```

Comment:
Yes, I've added more getter because the getters included give me errors, but i think that we could implement a "score" for each getter and sorter it for number of fails or even response time.

I can work on it later